test/scenarios-bootc/presubmits/el96-src@ai-model-serving-online.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# Currently, RHOAI is only available for x86_64
+check_platform() {
+    local -r record_junit=${1:-false}
+
+    if [[ "${UNAME_M}" =~ aarch64 ]]; then
+        if "${record_junit}"; then
+            record_junit "setup" "scenario_create_vms" "SKIPPED"
```

Comment:
@agullon should we be more specific about the skip message here ? scenario_create_vms looks more generic, may be say like RHOAI is not supported on aarch64 so skipping ? WDYT ?

Reply:
@kasturinarra There's a comment about this on line 5, just before `check_platform()`. Do you think is that enough?

Reply:
If my understanding is correct, that will not be shown in the log right, later if some one tries to understand why the test is being skipped message we have may not be sufficient i believe,  WDYT or let us also wait for johns comment if you think we need to weigh in his input as well, thanks !!

Reply:
So the `record_junit()` function in scenario.sh does not really include a proper message:
```
record_junit() {
    local vmname="$1"
    local step="$2"
    local results="$3"
```

`SKIPPED` here is the type and gets transformed into `<skipped message="${results}" type="${step}-skipped" />` (${results} is the 3rd param, so effectively SKIPPED):
```
    case "${results}" in
...
        SKIP*)
        cat - >>"${JUNIT_OUTPUT_FILE}" <<EOF
<skipped message="${results}" type="${step}-skipped" />
EOF
```

Including a proper message would require some code changes to the function and that might be best to do in another PR